.dotnet.azure/sdk/openai/Azure.AI.OpenAI/src/Custom/Batch/AzureCreateBatchOperation.Protocol.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.ClientModel;
+using System.ClientModel.Primitives;
+
+#nullable enable
+
+namespace Azure.AI.OpenAI.Batch;
+
+/// <summary>
+/// A long-running operation for executing a batch from an uploaded file of 
+/// requests.
+/// </summary>
+public partial class AzureCreateBatchOperation : CreateBatchOperation
```

Comment:
If these need to be public, should they be marked with [Experimental]?  Or is [Experimental] applied to a namespace so this gets this from an outer scope?

Reply:
made internal